cpp/velox/compute/VeloxBackend.cc

```diff
@@ -195,7 +195,7 @@ void VeloxBackend::initJolFilesystem() {
 }
 
 void VeloxBackend::initCache() {
-  if (backendConf_->get<bool>(kVeloxCacheEnabled, false)) {
+  if (backendConf_->get<bool>(kVeloxCacheEnabled, false) && !isDriver_) {
```

Comment:
Could just change from the caller code, something like:

```
if (!isDriver_) {
  initCache();
}
...
```

Would be enough and more readable.

Reply:
Done